From caeab26db0de8d0c008aaf28a1d2667fa5df9436 Mon Sep 17 00:00:00 2001
From: Manikanta Mylavarapu <mmanikan@qti.qualcomm.com>
Date: Sun, 1 Mar 2026 23:59:00 +0530
Subject: [PATCH] cpufreq: dt-platdev: add ipq5210 support

Change-Id: I928b319210c182c76b8a343686a4ac101362d2e8
Signed-off-by: Manikanta Mylavarapu <mmanikan@qti.qualcomm.com>
(cherry picked from commit 57bd541fe4408db8d1aecf0dca266ca16aab2239)
---
 drivers/cpufreq/cpufreq-dt-platdev.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/cpufreq/cpufreq-dt-platdev.c b/drivers/cpufreq/cpufreq-dt-platdev.c
index 7a03f0ae5486..12ac98a6f202 100644
--- a/drivers/cpufreq/cpufreq-dt-platdev.c
+++ b/drivers/cpufreq/cpufreq-dt-platdev.c
@@ -178,6 +178,7 @@ static const struct of_device_id blocklist[] __initconst = {
 	{ .compatible = "ti,am625", },
 	{ .compatible = "ti,am62a7", },
 
+	{ .compatible = "qcom,ipq5210", },
 	{ .compatible = "qcom,ipq5332", },
 	{ .compatible = "qcom,ipq5424", },
 	{ .compatible = "qcom,ipq8064", },
-- 
2.34.1

